src/libraries/Microsoft.Extensions.DependencyModel/ref/Microsoft.Extensions.DependencyModel.cs

```diff
@@ -4,6 +4,8 @@
 // Changes to this file must follow the https://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+using System.Diagnostics.CodeAnalysis;
```

Comment:
You shouldn't edit this file by hand. See [https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md) for how to update it automatically.

Reply:
@eerhardt Thanks for the tip! 
`dotnet msbuild /t:GenerateReferenceAssemblySource` didn't update nullable annotations though (only attributes). Am I missing something?

Reply:
Once you have the nullable annotations in place in the src project, `dotnet msbuild /t:GenerateReferenceAssemblySource` from the `src` directory should update the annotations.

@krwq @joperezr @Anipik - do either of you know why they wouldn't be updated in the `ref`?